p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -101,14 +101,15 @@ impl<E: Spawner + Rng + Clock + GClock + RuntimeMetrics, C: Signer> Actor<E, C>
         // We allow peers to be sent in any order when responding to a bit vector (allows
         // for selecting a random subset of peers when there are too many) and allow
         // for duplicates (no need to create an additional set to check this)
+        let my_public_key = self.crypto.public_key();
```

Comment:
This optimization does make sense if you want to keep this alone?

Reply:
Done.